content/en/docs/concepts/architecture/coordinated-leader-election.md

```diff
@@ -0,0 +1,56 @@
+---
+reviewers:
+- jpbetz
+title: Coordinated Leader Election
+content_type: concept
+weight: 200
+---
+
+<!-- overview -->
+
+{{< feature-state feature_gate_name="CoordinatedLeaderElection" >}}
+
+Kubernetes {{< skew currentVersion >}} includes an alpha feature that allows
+components to deterministically select a leader via Coordinated Leader Election.
```

Comment:
```suggestion
{{< glossary_tooltip text="control plane" term_id="control-plane" >}} components to
deterministically select a leader via _coordinated leader election_.
```
This doesn't look useful for clients using Lease that aren't part of the control plane, but maybe I'm wrong here?